### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by divyam3897**
Updated README and made following changes:
1.  put the logo at the top of the list
2. Add License like so
3. sure the all other badges are available.
Fixes https://github.com/sendgrid/open-source-library-data-collector/issues/78

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/open-source-library-data-collector?pullRequest=82) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/open-source-library-data-collector?pullRequest=82) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/open-source-library-data-collector?pullRequest=82) it.</sub>

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
+![SendGrid Logo](https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)
 [![Travis Badge](https://travis-ci.org/sendgrid/open-source-library-data-collector.svg?branch=master)](https://travis-ci.org/sendgrid/open-source-library-data-collector)
+[![BuildStatus](https://travis-ci.org/sendgrid/open-source-library-data-collector.svg?branch=master)](https://travis-ci.org/sendgrid/open-source-library-data-collector)
+[![Email Notifications Badge](https://dx.sendgrid.com/badge/python)](https://dx.sendgrid.com/newsletter/python)
+[![npm](https://img.shields.io/npm/l/express.svg)]()
```
Comment (mbernier) [inline README.md]: Please remove this line [npm]

**Comment by divyam3897** [comment]
@mbernier done 👍

**Comment by thinkingserious** [comment]
Hello @divyam3897,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**